src/main/java/pl/jablonskanycz/bakery/BakeryApplication.java

```diff
@@ -42,28 +42,29 @@ public static void main(String[] args) throws IOException {
         ProductRepository productRepository = new FiledBasedProductRepository();
         EmployeeRepository employeeRepository = new FileBasedEmployeeRepository();
 
-        Path addressPath = Path.of("src", "main", "resources", "ADDRESS.csv");
-        Path clientPath = Path.of("src", "main", "resources", "CLIENT.csv");
         Path employeePath = Path.of("src", "main", "resources", "EMPLOYEE.csv");
         Path productPath = Path.of("src", "main", "resources", "PRODUCT.csv");
 
         List<Address> addresses = addressRepository.getAll(); // to czyta z PLIKU
+//
+//
+//        for (Address address : addresses) {
+//            System.out.println(address);; // a to wkłada do LISTY
+//        }
 
+//        addressRepository.addAddress(Address.builder().latitude(16.89).longitude(21.43).build());
+//        System.out.println(addressRepository.findById(2));
 
-        for (Address address : addresses) {
-            System.out.println(address);; // a to wkłada do LISTY
-        }
-
-        addressRepository.addAddress(Address.builder().latitude(16.89).longitude(21.43).build());
-
-
-//        List<Client> clients = readClientsFrom(clientPath, addressRepository); // pytanie
-//        // (adresRepo)
+//        clientRepository.updateClient(new Client(2, "KAROL", "NOWAK", addressRepository.findById(2)), new Client(2, "KAROL", "NOWACKI", addressRepository.findById(2)));
+        clientRepository.deleteClient(new Client(2, "KAROL", "NOWAK", addressRepository.findById(2)));
```

Comment:
let's add one new feature so that one doesn't have to provide _all_ client's data - ideally just `id` should be enough! ;)